src/svgs/FolderIcon.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function FolderIcon({ onClick, isSelected }) {
```

Comment:
If we ever have to add another prop to these, I say we should just spread all the props on the SVG. But this is fine for now.